adal/tests/Test.ADAL.NET.UIAutomation/CoreMobileADALTests.cs

```diff
@@ -14,38 +14,132 @@ namespace Test.ADAL.NET.UIAutomation
     /// </summary>
 	public class CoreMobileADALTests
     {
+        private const string UiAutomationTestClientId = "3c1e0e0d-b742-45ba-a35e-01c664e14b16";
+        private const string MSIDLAB4ClientId = "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0";
+        private const string MSGraph = "https://graph.microsoft.com";
+        private const string Exchange = "https://outlook.office365.com/";
+        private const string UiAutomationTestResource = "ae55a6cc-da5e-42f8-b75d-c37e41a1a0d9";
+        private const string AcquireTokenID = "acquireToken";
+        private const string AcquireTokenSilentID = "acquireTokenSilent";
+        private const string clientIdEntryID = "clientIdEntry";
+        private const string resourceEntryID = "resourceEntry";
+        private const string secondPageID = "secondPage";
+        private const string ClearCacheID = "clearCache";
+        private const string SaveID = "saveButton";
+        private const string WebUPNInputID = "i0116";
+        private const string AdfsV4WebPasswordID = "passwordInput";
+        private const string AdfsV4WebSubmitID = "submitButton";
+        private const string WebPasswordID = "i0118";
+        private const string WebSubmitID = "idSIButton9";
+        private const string TestResultID = "testResult";
+        private const string TestResultSuccsesfulMessage = "Result: Success";
```

Comment:
put all of these in a helper class which both the ADAL and the MSAL test can reference